.github/workflows/core-build-test.yml

```
@@ -0,0 +1,133 @@
+              - '!.github/dependabot.yml'
+              - '!.github/CODEOWNERS'
+              - '!.github/workflows/**'
+              - '!examples/**'
+              - '!vm2/move-examples/**'
+              - '!scripts/**'
+              - '!kube/**'
+              - '!docker/**'
+              - '**'
+      - name: run fmt check
+        if: steps.changes.outputs.code == 'true'
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)